README.md

```diff
@@ -1,75 +1,24 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# README
```

Comment:
- [ ] Kindly format your README using the Microverse template for more professional appearance